src/helpers/cloneChildren.ts

```diff
@@ -89,7 +89,16 @@ function getFormableComponentProperties(errors, fieldErrors, onSubmit, onChange)
 export function createFormableRule(errors = [], fieldErrors = {},
```

Comment:
readability could probably be improved here if all the arguments do not fit on one line. 

```js
export function createFormableRule(
    errors = [], 
    fieldErrors = {},
    onSubmit = identity, 
    onChange = identity
) {
```